libs/localization/src/lib/en.json

```diff
@@ -1300,8 +1300,9 @@
       "CorrectPredictions": "Correct predictions",
       "GlobalExplanation": "Aggregate feature importance",
       "IncorrectPredictions": "Incorrect predictions",
-      "IndividualFeature": "Select a datapoint by clicking on a datapoint (or multiple datapoints) in the table to view their local feature importance values (local explanation) and individual conditional expectation (ICE) plot below. For datasets with more than 5000 datapoints, the view is a random subsample to enable easy exploration.",
-      "LocalExplanation": "Individual feature importance"
+      "IndividualFeature": "Select a datapoint by clicking on a datapoint (up to 5 datapoints) in the table to view their local feature importance values (local explanation) and individual conditional expectation (ICE) plot below. For datasets with more than 5000 datapoints, the view is a random subsample to enable easy exploration.",
+      "LocalExplanation": "Individual feature importance",
+      "SelectionLimit": "Up to 5 datapoints can be selected at this time."
```

Comment:
We can't make it 10?  Did PM decide on the number 5?

Reply:
Yes this number 5 is from PM. We can collect feedback and update the number. But I feel 5 is a good number for now because otherwise feel like ICE plot will contain too many lines
![image](https://user-images.githubusercontent.com/91754176/161146385-949318b6-a4ab-4694-bce2-571e64a378fa.png)